component/core/src/main/java/org/exoplatform/social/core/space/impl/SpaceTemplateServiceImpl.java

```
@@ -182,4 +267,24 @@ public void start() {
   public void stop() {
 
   }
+
+  private Identity getIdentity(String userId) {
+    IdentityRegistry identityRegistry = CommonsUtils.getService(IdentityRegistry.class);
+    OrganizationService organizationService = CommonsUtils.getService(OrganizationService.class);
+    Identity identity = identityRegistry.getIdentity(userId);
+    if (identity == null) {
+      Collection<Membership> memberships = new ArrayList<>();
+      try {
+        memberships = organizationService.getMembershipHandler().findMembershipsByUser(userId);
+      } catch (Exception e) {
+        LOG.error("Can't get user '{}' memberships", userId, e);
```

We should throw this exception and manage it in UI to display an error

---

ok